test/metabase/test/data/mbql_query_impl.clj

```diff
@@ -10,13 +10,14 @@
 (set! *warn-on-reflection* true)
 
 (defn- token->sigil [token]
-  (when-let [[_ sigil] (re-matches #"^([$%*!&]{1,2}).*[\w/]$" (str token))]
+  (when-let [[_ sigil] (re-matches #"^([$%*!&]{1,2}).*[\w/]$" (name token))]
     sigil))
 
 (defmulti ^:private parse-token-by-sigil
   {:arglists '([source-table-symb token])}
   (fn [_ token]
```

Comment:
I believe the ability to use keywords can make our tests have a more "data-y" feel and I think this change is compatible since `name` on a token and a non-ns-qualified keyword produces the same output.